src/lib/Scenes/SavedSearchAlert/Components/Form.tsx

```diff
@@ -66,7 +66,7 @@ export const Form: React.FC<FormProps> = (props) => {
         <InputTitle>Filters</InputTitle>
         <Flex flexDirection="row" flexWrap="wrap" mt={1} mx={-0.5}>
           {pills.map((pill, index) => (
-            <Pill testID="alert-pill" m={0.5} key={`filter-label-${index}`}>
+            <Pill testID="alert-pill" size="xs" rounded m={0.5} key={`filter-label-${index}`}>
```

Comment:
Could you remove `rounded` from this Pill? This is supposed to be square according to design. More info can be found on [this slack thread](https://artsy.slack.com/archives/C9SATFLUU/p1630572009268600)
```suggestion
            <Pill testID="alert-pill" size="xs" m={0.5} key={`filter-label-${index}`}>
```